Example/Auth/Sample/MainViewController.m

```diff
@@ -2458,32 +2516,41 @@ - (void)signInWithPhoneNumber {
   }];
 }
 
+/** @fn signInWithPhoneNumberRecaptchaWithString
+    @brief Allows sign in with phone number using reCAPTCHA and completion.
+    @param phoneNumber Number pass in string for automation.
+ */
+- (void)signInWithPhoneNumberRecaptchaWithString:(NSString *_Nullable)phoneNumber
+                                completion:(void(^)(NSError *_Nullable))completion {
+  [self showSpinner:^{
+    [[AppManager phoneAuthProvider] verifyPhoneNumber:phoneNumber
+                                           UIDelegate:nil
+                                           completion:^(NSString *_Nullable verificationID,
+                                                        NSError *_Nullable error) {
+      [self hideSpinner:^{
+        if (error) {
+          [self logFailure:@"failed to send verification code" error:error];
+          [self showMessagePrompt:error.localizedDescription];
```

Comment:
In case of error, you need to call the completion block with the error before returning.

Example:

```
completion(error);
return;
```